TeachingRecordSystem/src/TeachingRecordSystem.Api/V3/VNext/Controllers/PersonsController.EwcWales.cs

```diff
@@ -8,7 +8,7 @@
 namespace TeachingRecordSystem.Api.V3.VNext.Controllers;
 
 [Route("persons")]
-public class PersonsController : ControllerBase
+public partial class PersonsController : ControllerBase
```

Comment:
This is moved into a partial class because it's likely to be stuck in `VNext` for a while and it makes promoting other endpoints simpler if this is in its own file.